native_shortuuid/admin.py

```diff
@@ -18,7 +19,8 @@ def is_valid_shortuuid(self, search_term):
 
     def get_search_fields(self, request):
         search_fields = list(super().get_search_fields(request) or [])
-        if getattr(django.conf.settings, 'ADMIN_AUTO_EXTRACT_UUID_SEARCH_FIELDS', True):
+        if getattr(django.conf.settings, 'ADMIN_AUTO_EXTRACT_UUID_SEARCH_FIELDS', True) \
+                or self.admin_auto_extract_uuid_search_fields:
             self.search_uuid_fields = []
             for field in self.model._meta.fields:
```

Comment:
we should loop through search_fields, if field is in model fields check for its type, if not, check if it ends with uuid. Check to see if there is a way to resolve field name with foreign key reference to field instance.